bessctl/conf/samples/gurobi.bess

```diff
@@ -0,0 +1,138 @@
+#
```

Comment:
Can you clarify on the file name 'gurobi.bess'?
How is it related to Gurobi?

Reply:
Its a legacy name. Changed it.